stanford-sw/src/edu/stanford/Item.java

```diff
@@ -94,7 +94,8 @@ public Item(DataField f999, String recId) {
 					|| StanfordIndexer.SKIPPED_LOCS.contains(homeLoc)
 					|| itemType.equals("EDI-REMOVE")
 					// SW-849 skip PHYSICS items that aren't location PHYSTEMP
-					|| (library.equals("PHYSICS") && (!homeLoc.equals("PHYSTEMP") && !currLoc.equals("PHYSTEMP"))))
+					|| (library.equals("PHYSICS") && (!homeLoc.equals("PHYSTEMP") && !currLoc.equals("PHYSTEMP")))
+          || StanfordIndexer.CLOSED_LIBS.contains(library))
```

Comment:
In Atom, In the Packages menu, I have a "whitespace" thing I can use to convert tabs to whitsepace.  It seems to miss a few tabs sometimes.  I'm not sure if that came with Atom or if I got an add on.